internal/webservice/routes.go

```diff
@@ -126,7 +126,7 @@ func (ws *WebService) init(ctx context.Context) {
 	)
 	service.Route(
 		service.GET(routeNodeUtilization).
-			To(ws.getNodeUtilizations).
+			To(ws.getNodesUtils).
 			Produces(restful.MIME_JSON).
 			Writes([]dao.PartitionNodesUtilDAOInfo{}).
```

Comment:
You probably need to update this as well for documentation purposes. It writes `model.NodeUtil` now